powerlevel9k.zsh-theme

```diff
@@ -770,11 +769,13 @@ prompt_dir() {
 
   local -i fake_first=0
   local delim=${POWERLEVEL9K_SHORTEN_DELIMITER-$'\u2026'}
+  local -i shortenlen=${POWERLEVEL9K_SHORTEN_DIR_LENGTH:--1}
+  local -i d=${POWERLEVEL9K_SHORTEN_DELIMITER_LENGTH:-$#POWERLEVEL9K_SHORTEN_DELIMITER}
```

Comment:
FYI: There is a small big here: the default value of `d` should be `$#delim` (also note that `delim` may get changed below). I've fixed it in https://github.com/romkatv/powerlevel10k/commit/3be988cae87a22a4f405ca088a5fe6e06d94e445. I also changed the default to `_p9k_prompt_length $delim`. This gives the length the delimiter will occupy when printed to the terminal.

Reply:
Yeah, you're right. Nice!